analyser/panako_processor.py

```
@@ -117,114 +117,54 @@ def upload_sample_fingerprints(sample_id, fingerprints):
-    Best-effort per sample: a single sample failure is logged and skipped so
-    one bad sample doesn't poison the whole pass. Returns (ok_count, fail_count).
-
-    Note: until the new persist endpoint is deployed server-side, match
-    results are written to SAMPLE_MATCH_RESULTS_FILE (gitignored). Once the
-    backend is deployed, switch the per-sample call to the POST endpoint and
-    drop the local file.
-    """
-    print(f"[scoring] {reason} — listing samples with fingerprints…")
-    try:
-        samples, endpoint_deployed = list_samples_with_fingerprint()
+        res = requests.post(endpoint, headers=auth_header(), json={})
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)